src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByEmailController.cs

```diff
@@ -66,6 +66,17 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, SearchByEmailSubmi
 
         var relationshipByEmail = await _outerApiClient.GetRelationshipByEmail(submitModel.Email!, ukprn, cancellationToken);
 
+        if (relationshipByEmail.Paye != null)
+        {
+            var getRequestResponse = await _outerApiClient.GetRequest(ukprn, relationshipByEmail.Paye, cancellationToken);
+            if (getRequestResponse.ResponseMessage.IsSuccessStatusCode)
+            {
+                sessionModel.Paye = relationshipByEmail.Paye;
+                _sessionService.Set(sessionModel);
+                return RedirectToRoute(RouteNames.AddEmployerInvitationAlreadySent, new { ukprn });
+            }
```

Comment:
This logic should exist in the outer api... usually we only have one call to the outer from web

Reply:
This isn't a new call I've added, it was already there and I changed the placement to work with the logic change.  I'll have a look through and see what I can do

Reply:
I'll take a look

Reply:
yes, you where right so will change it, however, I need to work out if it's a request against the email entered, or against another email (with same paye) and send to the right shutter page.  Just doing that rework.

Reply:
There is only one common shutter page I think